apps/site/theme.config.tsx

```diff
@@ -26,6 +26,9 @@ const config: DocsThemeConfig = {
     defaultTheme: 'light',
   },
   darkMode: false,
+  toc: {
```

Comment:
What does it do? 👀

Reply:
It removes the right sidebar (toc =  table of content): https://nextra.site/docs/docs-theme/theme-configuration#toc-sidebar